src/components/dashboard/FaceVerification/components/UnrecoverableError.js

```diff
@@ -26,26 +26,37 @@ if (Platform.OS === 'web') {
   Image.prefetch(illustration)
 }
 
-const UnrecoverableError = ({ styles, exception, screenProps }) => {
+const UnrecoverableError = ({ styles, exception, attemptsHistory, screenProps }) => {
   const [, hideDialog, showErrorDialog] = useDialog()
+  const { navigateTo, goToRoot, push } = screenProps
 
   const sdkStatus = get(exception, 'code')
   const isLicenseIssue = [InvalidDeviceLicenseKeyIdentifier, LicenseExpiredOrInvalid].includes(sdkStatus)
 
-  const onContactSupport = useOnPress(() => screenProps.navigateTo('Support'), [screenProps])
-  const onDismiss = useOnPress(() => screenProps.goToRoot(), [screenProps])
+  const onContactSupport = useOnPress(() => navigateTo('Support'), [navigateTo])
+  const onDismiss = useOnPress(() => goToRoot(), [goToRoot])
 
   useEffect(() => {
+    const { message } = exception
+
+    // logging that unrecoverable error screen shown
+    log.error(
```

Comment:
this feels like a duplicate, as the error is fired in trackedattempts
you can restore the next log.error in case of license issue, though that should never happen